.dockerignore

```diff
@@ -3,8 +3,8 @@
 .pytest_cache
 .vagrant
 .vscode
-tests
-test_e2e
+#tests
+#test_e2e
```

Comment:
Rather than commenting these out, let's just remove them; otherwise it's not clear if this is meant to be temporary or permanent

Reply:
thanks